packages/integration-tests/src/test-project-setup/test_project_base.ts

```diff
@@ -144,4 +149,40 @@ export abstract class TestProjectBase {
 
     assert.ok(clientConfigStats.isFile());
   }
+
+  /**
+   * Verify deployed client outputs
+   */
+  async assertDeployedClientOutputs(backendId: BackendIdentifier) {
```

Comment:
should this just be part of `assertPostDeployment`?

Reply:
I think we should leave this separate so it is not coupled with other e2e tests and potentially fail before other checks in `assertPostDeployment`.